pom.xml

```diff
@@ -128,11 +113,6 @@
             <artifactId>jakarta.xml.bind-api</artifactId>
             <version>2.3.3</version>
         </dependency>
-        <dependency>
-            <groupId>jakarta.xml.ws</groupId>
-            <artifactId>jakarta.xml.ws-api</artifactId>
-            <version>2.3.3</version>
```

Comment:
can you set 
            <scope>provided</scope>
instead removing?
locally when I want to build project without this dependency I'm getting error that classes are missing